pkg/gatherers/clusterconfig/const.go

```diff
@@ -14,9 +14,9 @@ const (
 var (
 	registryScheme = runtime.NewScheme()
 	// logTailLines sets maximum number of lines to fetch from pod logs
-	logTailLines = int64(100)
-	// logTailLinesLong sets the maximum number of lines to fetch from long pod logs
-	logTailLinesLong = int64(400)
+	logTailLines = int64(4000)
```

Comment:
@tremes instead get the full log I'm getting a pretty reasonable amount of lines, which IMHO, would give us better performance. What do you think?

Reply:
4000 lines is quite a lot, so I believe we may want to add a size limit as well to avoid size problems caused by extremely long lines. It shouldn't be too difficult, right?

Reply:
It seems that we want to search using the full log... and also, we already use a size limit to the file.